input/postgres/relations.go

```diff
@@ -10,8 +10,10 @@ import (
 	"github.com/pganalyze/collector/state"
 )
 
-const relationsSQLDefaultOptionalFields = "0"
-const relationsSQLpg93OptionalFields = "c.relminmxid"
+const relationsSQLDefaultMxidFields = "1"
```

Comment:
[note for reviewer]
This default value was previously `0`, but I'm going to change it to `1`, as I've been observing that it is actually the right default value. The database side was already having 1 as a default value, interestingly.